setup.py

```diff
@@ -0,0 +1,54 @@
+# make code as python 3 compatible as possible
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import subprocess
+import setuptools
+import os.path
+import sys
+
+from setuptools.command.develop import develop
+from setuptools.command.install import install
+
+
+# Build anltr files on installation
+#   this is such a mess... it looks like there are
+#   no common steps to develop and install
+
+class AntlrDevelopCommand(develop):
+    def run(self):
+        compile_grammar()
+        develop.run(self)
+
+class AntlrInstallCommand(install):
+    def run(self):
+        compile_grammar()
+        install.run(self)
+
+def compile_grammar():
+    here = os.path.dirname(__file__) or '.'
+    package_dir = os.path.join(here, 'latex2sympy')
+    subprocess.check_output(['antlr4',  'PS.g4', '-o', 'gen'], cwd=package_dir)
+
+if sys.version_info.major == 3:
+    REQUIRES = ['antlr4-python3-runtime',  'sympy']
+else:
+    REQUIRES = ['antlr4-python2-runtime',  'sympy']
+
+setuptools.setup(
+    name='latex2sympy',
+    version=0.1,
```

Comment:
Should be a string, and should _ideally_ appear in exactly one place (i.e. `latex2sympy/_version.py`) and be read here.

Reply:
Well, it already appears in one place: `setup.py`. But adding a `_version.py` (or perhaps `__version__` in the package `__init__.py` and pulling this in would make a version number accessible from code. I can see the value of this when trying to work out exactly what version of this package is running. On the other hand, there is a price in terms of discoverability and I don't imagine this package particularly changing a great deal.